packages/web-client/app/utils/web3-strategies/layer-1-connection-manager.ts

```diff
@@ -439,20 +439,18 @@ class WalletConnectConnectionStrategy extends ConnectionStrategy {
         qrcodeModal: WalletConnectQRCodeModal,
       };
     }
+    let hubConfigApi = new HubConfig(config.hubURL);
```

Comment:
for the near term solution aren't we going to have 2 endpoints--one that the web client uses that has a referrer URL limitation, and one that everyone else uses that does not? if so, I think we'll need to get this endpoint from a different place (maybe the environments.js files?)